mm2src/coins/eth/eth_rpc.rs

```diff
@@ -155,6 +157,15 @@ impl EthCoin {
 
     /// Get balance of given address
     pub(crate) async fn balance(&self, address: Address, block: Option<BlockNumber>) -> Result<U256, web3::Error> {
+        if self.is_tron() {
+            // TODO use Tron client
+            let sun = U256::zero();
+            warn!(
+                "Using stub implementation for Tron address_balance for {}, returning {sun} SUN",
+                format!("{:?}", TronAddress::from(&address)),
+            );
+            return Ok(sun);
+        }
```

Comment:
don't we wanna do such a check in most places since we now merged CoinType::Eth into CoinType::TRX?